build.gradle

```diff
@@ -500,6 +500,14 @@ project('spring-integration-mail') {
 	[compileJava,compileTestJava]*.options*.compilerArgs = ["${xLintArg},-path"]
 }
 
+project('spring-integration-mock') {
```

Comment:
Well, I decided to leave `SI-test-support` package as is.
And in a new module I just did a couple sub-packages.
So, now we have there `o.s.i.test.context` and `o.s.i.test.mock`. With the the new module is just like a sub-module of the existing `support` structure.

Does it make sense?